write-api-service/src/main/java/io/ga4gh/reference/api/GitHubBuilder.java

```diff
@@ -100,7 +101,8 @@ public boolean createRepo(String organization, String repo) {
             // was not able to create the repo
             return false;
         } catch (IOException e) {
-            throw new RuntimeException(e);
+            LOG.error(e.getMessage());
+            throw new RuntimeException("Could not create GitHub repository.  Check your GitHub token.");
```

Comment:
I think I put LOG.error for that one just out of habit, the e.getMessage() returned something that was but very unhelpful like "error" so i did not bother putting it in the constructor.